flux-fhir-analysis/src/lib.rs

```diff
@@ -142,6 +143,11 @@ fn item_bounds(
     Ok(rty::EarlyBinder(conv::conv_opaque_ty(genv, local_id.to_def_id(), opaque_ty, &wfckresults)?))
 }
 
+// TODO(RJ): using `Vec` instead of `List` due to some inscrutable rustc error
+fn refine_params_of(genv: &GlobalEnv, local_id: LocalDefId) -> QueryResult<Vec<rty::RefineParam>> {
+    Ok(generics_of(genv, local_id)?.refine_params)
+}
```

Comment:
We don't need this as a query anymore. I was thinking adding the method directly to `GlobalEnv`